### PR TITLE
Adjust command for integration-tests in Prow

### PR DESCRIPTION
**Pull request by fridex**
## Related Issues and Dependencies

Fixes: https://github.com/thoth-station/integration-tests/issues/125

## This introduces a breaking change

- [x] No

### CODE REVIEW

File: prow/overlays/cnv-prod/config.yaml [hunk 1]
```diff
@@ -305,10 +305,21 @@ periodics:
...
+            - name: MAIL_REPORT
+              value: "0"
+            - name: THOTH_USER_API_HOST
               value: api.moc.thoth-station.ninja
+            - name: THOTH_MANAGEMENT_API_HOST
+              value: management.moc.thoth-station.ninja
+            - name: THOTH_AMUN_API_HOST
+              value: amun.moc.thoth-station.ninja
+            - name: THOTH_MANAGEMENT_API_SECRET
+              value: ""
```
Comment (fridex) [inline prow/overlays/cnv-prod/config.yaml:322]: Do we want to have this setup?

Comment (harshad16) [reply prow/overlays/cnv-prod/config.yaml:322]: oh yes we do have set these up. 
same as the stage as of now :see_no_evil:

Comment (harshad16) [reply prow/overlays/cnv-prod/config.yaml:322]: we can have these value set by configmap and secrets. As this component is not in thoth namespaces, so we can have these setup in prow namespace too explicitly.

Comment (fridex) [reply prow/overlays/cnv-prod/config.yaml:322]: Do we want to keep running these tests in Prow? I wonder if we could rather deploy integration-tests in a namespace running internally and point it to the prod deployment. It is a manner of a new overlay in integration-tests with adjusted setup. We will not need to maintain integration-tests in prow and we reuse the same logic we already have for test and stage environments (without a need to provide secrets here, have periodic e-mail reports). Does Prow give us any advantage here?

Comment (harshad16) [reply prow/overlays/cnv-prod/config.yaml:322]: I m in favor of running a cronjob, which sends out emails. It has an advantage added to it.
prow basically is just executing the cronjob, so I don't see any added advantage.

Comment (goern) [reply prow/overlays/cnv-prod/config.yaml:322]: prow advantage I see is, that it keeps all the logs around, and that we declare all the things wrt prod in one place.

File: prow/overlays/cnv-prod/config.yaml [hunk 2]
```diff
@@ -305,10 +305,21 @@ periodics:
         - image: quay.io/thoth-station/integration-tests:v0.5.0
           imagePullPolicy: Always
           command:
-            - ./e2e-test.sh
+            - python3
```
Comment (goern) [inline prow/overlays/cnv-prod/config.yaml:308]: who is going to install the dependencies?

Comment (goern) [reply prow/overlays/cnv-prod/config.yaml:308]: see https://github.com/thoth-station/integration-tests/pull/136

Comment (fridex) [reply prow/overlays/cnv-prod/config.yaml:308]: > who is going to install the dependencies?

They should be already present in the container as part of the s2i build process.

Comment (goern) [reply prow/overlays/cnv-prod/config.yaml:308]: ja, .makes sense, thx!

**Pull request merged**